internal/integration/base/cli.go

```diff
@@ -38,22 +38,22 @@ func (cliSuite *CLISuite) DiscoverNodes() []string {
 	return nil
 }
 
-func (cliSuite *CLISuite) buildOsctlCmd(args []string) *exec.Cmd {
+func (cliSuite *CLISuite) buildCLICmd(args []string) *exec.Cmd {
 	// TODO: add support for calling `talosctl config endpoint` before running talosctl
 
 	args = append([]string{"--talosconfig", cliSuite.TalosConfig}, args...)
 
 	return exec.Command(cliSuite.TalosctlPath, args...)
 }
 
-// RunOsctl runs talosctl binary with the options provided
-func (cliSuite *CLISuite) RunOsctl(args []string, options ...RunOption) {
-	Run(&cliSuite.Suite, cliSuite.buildOsctlCmd(args), options...)
+// RunCLI runs talosctl binary with the options provided
```

Comment:
More generic naming so that _if_ we ever change the cli name again, we don't have to update everywhere.